backend/src/api/routes/auth/oauth.routes.ts

```
@@ -293,7 +289,11 @@ router.get('/:provider', async (req: Request, res: Response, next: NextFunction)
       expiresIn: '1h', // Set expiration time for the state token
     });
 
-    const authUrl = await authService.generateOAuthUrl(validatedProvider, state);
+    const authUrl = await authService.generateOAuthUrl(
+      validatedProvider,
+      state,
+      additionalParams
+    );
```

ESLint

[error] Replace `⏎······validatedProvider,⏎······state,⏎······additionalParams⏎····` with `validatedProvider,·state,·additionalParams`

(prettier/prettier)